### ISSUE TITLE
Add mozconfig file

### ISSUE THREAD

**Issue by ClassicNick** [ClassicNick/fx36oldwin#4]
This is necessary because builds will certainly fail without it, and certain options need to be disabled in order to build. This includes: accessibility, tests, ipc, crashreporter, and updater.

**Comment by ClassicNick** [comment]
.mozconfig file has been added, so builds should now succeed.
